e2e/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "firebase-smoke-test",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "setup": "node test-setup.js",
+    "test": "yarn setup && karma start --compat --exp",
```

Comment:
Maybe worth running tests in Nodejs as well

Reply:
Seems like a good idea, probably should be an additional PR.